new-log-viewer/package-lock.json

```diff
@@ -27,7 +27,6 @@
         "babel-loader": "^9.1.3",
```

Comment:
Changes in this file are experimental and not at all related to the pull request at https://github.com/y-scope/yscope-log-viewer/pull/57